FirstSolution/Task4Germany/Berlin.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace Task4Germany
+{
+    public class Berlin
+    {
+        private static double thousandOfInhabitants = 3645.5;
```

Comment:
Is it possible that the number of inhabitants is double?

Reply:
Changed